.github/workflows/pullrequest-stale.yml

```diff
@@ -0,0 +1,25 @@
+name: Stale
+on:
+  schedule:
+    - cron: "0 * * * *"
```

Comment:
I do not think we need to run this [every hour](https://crontab.guru/#0_*_*_*_*)? I'd suggest daily sometime in the night: `0 3 * * *`.

Reply:
I agree, that it does not need to run every hour, but I would prefer a run during the day, since otherwise people get notifications/emails in the middle of the night and I think we could avoid that. I personally prefer to get a notification during the day so I can act right away. What is your opinion on this?

Reply:
Sounds okay to me. :)

Reply:
Fixed 🙂